src/current/v24.3/manage-logical-data-replication.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Manage Logical Data Replication
+summary: Manage the tables that are part of the logical data replication stream.
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+Once you have **logical data replication (LDR)** running, you will need to track and manage certain parts of the job:
+
+- [Conflict resolution](#conflict-resolution): As changes to a table replicate from the source to the destination cluster, there can be conflicts during some operations that the job will handle with conflict resolution. When LDR is started, the job creates a [_dead letter queue (DLQ)_](#dead-letter-queue-dlq) table with each replicating table. LDR will send any unresolved conflicts to the DLQ, which you should monitor as LDR continues to replicate changes between the source and destination clusters. 
```

Comment:
could we separate DLQ from conflict resolution? the DLQ also manages "conflicts" but its actually a different definition of "conflict," like a secondary index "conflict"

Reply:
Actually nvm. I see how you talked about it later.